test_configs/windows_process_events/anonymizer_settings.json

```diff
@@ -0,0 +1,18 @@
+# Fields to be set in anonymizers.py
+# Un-string the provider_map field values while setting them in anonymizers.py
+{
+    "provider_map": {
+        "host.hostname": "self.faker.word",
+        "host.ip": "self.faker.ipv4",
+        "labels.account_id": "self.faker.ssn",
+        "labels.endpoint_id": "self.faker.ssn",
+        "observer.name": "self.faker.word",
+        "user.domain": "self.faker.word",
+        "user.name": "self.faker.word"
+    },
+    "high_cardinality_fields" = {},
+    "user_regexes" = {
+        "regex_1": "(Users)\\\\([^\\\\]+)",
+        "regex_2": "(Users)\\/([^\\/]+)"
```

Comment:
Can you please provide a more user-friendly name for these regex

Reply:
I have changed them to `user_dir_1` and `user_dir_2` if that helps.